portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -10,9 +27,40 @@
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
+  private final Date startTime = new Date();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    // Calculate server stats
+    Date currentTime = new Date();
+
+    // Convert data to JSON
+    MarcoStats marcoStats = new MarcoStats(20, "Marco");
+    String json = convertToJsonUsingGson(marcoStats);
```

Comment:
nit: `json` is too generic a variable name. Try something like `marcoStatsJson`.